Bug Report: win32 app missing Windows11_24H2 as requirement

## Description

I'm trying to set `minimum_supported_windows_release` to `Windows11_24H2`. This value is currently not in the validation set.


### Steps to Reproduce

Try to plan a "microsoft365_graph_beta_device_and_app_management_win32_app" with `minimum_supported_windows_release` set to `Windows11_24H2`

```
resource "microsoft365_graph_beta_device_and_app_management_win32_app" "notepad_v8_9_1" {
  allow_available_uninstall = true

  app_installer = {
    installer_file_path_source = "C:\\temp\\Software\\Output\\npp.8.9.1.Installer.x64.intunewin"
  }

  // Optional
  app_icon = {
    icon_url_source = "https://upload.wikimedia.org/wikipedia/commons/f/f5/Notepad_plus_plus.png"
  }

  // Optional
  categories = [
    "Business",
    "Productivity",
  ]

  description     = "Notepad++ v8.9.1 DES"
  publisher       = "Don Ho Pub"
  developer       = "Don Ho Dev"
  display_name    = "Notepad++ v8.9.1 DIS"
  display_version = "8.9.1"
  file_name       = "npp.8.9.1.Installer.x64.intunewin"
  information_url = "https://notepad-plus-plus.org/"

  owner = "Jan"
  notes = "Notes"

  allowed_architectures             = ["x64"]
  minimum_supported_windows_release = "Windows11_24H2"

  install_experience = {
    device_restart_behavior = "allow"
    max_run_time_in_minutes = 60
    run_as_account          = "system"
  }

  setup_file_path        = "Invoke-AppDeployToolkit.exe"
  install_command_line   = "Invoke-AppDeployToolkit.exe -DeploymentType Install"
  uninstall_command_line = "Invoke-AppDeployToolkit.exe -DeploymentType Uninstall"

  # Detection Rules
  rules = [
    {
      rule_type                  = "detection"
      rule_sub_type              = "file_system"
      path                       = "C:\\Program Files\\Notepad++"
      file_or_folder_name        = "notepad++.exe"
      check_32_bit_on_64_system  = false
      file_system_operation_type = "version"
      lob_app_rule_operator      = "greaterThanOrEqual"
      comparison_value           = "8.9.1"
    },
  ]

  return_codes = [
    {
      return_code = 0
      type        = "success"
    },
    {
      return_code = 1707
      type        = "success"
    },
    {
      return_code = 3010
      type        = "softReboot"
    },
    {
      return_code = 1641
      type        = "hardReboot"
    },
    {
      return_code = 1618
      type        = "retry"
    }
  ]

}
```

### Expected Behavior

It should be possible to set `minimum_supported_windows_release` to `Windows11_24H2`. As this can be configured using the Intune GUI.

### Environment

- **OS**: Windows
- **Browser**: not used
- **Terraform Version**: Tested with Terraform 1.14.8 and OpenTofu 1.11.6
- **Provider Version**: 0.54.0-alpha
- **Other dependencies**: no


### Logs & Console Output

```
2026-06-14T14:57:19.814+0200 [ERROR] vertex "microsoft365_graph_beta_device_and_app_management_win32_app.notepad_v8_9_1" error: Invalid Attribute Value Match
╷
│ Error: Invalid Attribute Value Match
│ 
│   with microsoft365_graph_beta_device_and_app_management_win32_app.notepad_v8_9_1,
│   on app.tf line 31, in resource "microsoft365_graph_beta_device_and_app_management_win32_app" "notepad_v8_9_1":
│   31:   minimum_supported_windows_release = "Windows11_24H2"
│ 
│ Attribute minimum_supported_windows_release value must be one of: ["Windows11_23H2" "Windows11_22H2" "Windows11_21H2" "Windows10_22H2" "Windows10_21H2" "21H1" "2H20" "2004" "1909" "1903"
│ "1809" "1803" "1709" "1703" "1607"], got: "Windows11_24H2"
```


### Priority

How critical is this issue? Choose one of the following:
- Medium (Needs to be fixed but not urgent)

--- a/internal/services/resources/device_and_app_management/graph_beta/win32_app/resource.go
+++ b/internal/services/resources/device_and_app_management/graph_beta/win32_app/resource.go
@@ -616,7 +616,7 @@ func (r *Win32LobAppResource) Schema(ctx context.Context, req resource.SchemaReq
 				Required:            true,
 				MarkdownDescription: "The value for the minimum supported windows release.",
 				Validators: []validator.String{
-					stringvalidator.OneOf("Windows11_23H2", "Windows11_22H2", "Windows11_21H2", "Windows10_22H2", "Windows10_21H2", "21H1", "2H20", "2004", "1909", "1903", "1809", "1803", "1709", "1703", "1607"),
+					stringvalidator.OneOf("Windows11_24H2", "Windows11_23H2", "Windows11_22H2", "Windows11_21H2", "Windows10_22H2", "Windows10_21H2", "21H1", "2H20", "2004", "1909", "1903", "1809", "1803", "1709", "1703", "1607"),
 				},
 			},
 			"display_version": schema.StringAttribute{

